eng/helix/content/runtests.sh

```diff
@@ -30,6 +28,17 @@ RED="\033[0;31m"
 YELLOW="\033[0;33m"
 MAGENTA="\033[0;95m"
 
+echo "Ensure $DOTNET_ROOT/shared directories exist..."
+mkdir -p $DOTNET_ROOT/shared/Microsoft.AspNetCore.App
+mkdir -p $DOTNET_ROOT/shared/Microsoft.NETCore.App
+
+echo "Symlink sdk: ln -s $HELIX_CORRELATION_PAYLOAD/dotnet/sdk $DOTNET_ROOT/sdk"
+ln -s $HELIX_CORRELATION_PAYLOAD/dotnet/sdk $DOTNET_ROOT/sdk
+echo "Symlink : ln -s $HELIX_CORRELATION_PAYLOAD/dotnet/shared/Microsoft.AspNetCore.App/*  $DOTNET_ROOT/shared/Microsoft.AspNetCore.App/"
+ln -s $HELIX_CORRELATION_PAYLOAD/dotnet/shared/Microsoft.AspNetCore.App/*  $DOTNET_ROOT/shared/Microsoft.AspNetCore.App/
+echo "Symlink : ln -s $HELIX_CORRELATION_PAYLOAD/dotnet/shared/Microsoft.NETCore.App/*  $DOTNET_ROOT/shared/Microsoft.NETCore.App/"
+ln -s $HELIX_CORRELATION_PAYLOAD/dotnet/shared/Microsoft.NETCore.App/*  $DOTNET_ROOT/shared/Microsoft.NETCore.App/
```

Comment:
Curious: What about dotnet/host/, dotnet/packs/, and dotnet/templates/❔ Did tests work because they're not found using `$DOTNET_ROOT`❔

Reply:
I started with symlinking everything, but it didn't look like we needed all of the other stuff, so I got rid of it... I figure its better to be explicit so we know exactly what we depend on